public/editor/scripts/main.js

```diff
@@ -47,6 +47,7 @@ require(["jquery", "bowser", "analytics"], function($, bowser, analytics) {
   function onError(err) {
     console.error("[Bramble Error]", err);
     $("#spinner-container").addClass("loading-error");
+    analytics.event({ category : analytics.eventCategories.TROUBLESHOOTING, action : "Editor loading error (black Screen)" });
```

Comment:
You could try `err.toString()` and just see how it goes.  The worst thing that will happen is we'll crash, but the editor is already crashed, so should be fine.